packages/core/test/directive.test.ts

```diff
@@ -16,4 +16,19 @@ describe('Directive test', () => {
     expect(query().ignoreReflex().toString())
       .toMatch(/@ignoreReflex \{/);
   });
+
+  it('`@recurse` directive no args', () => {
```

Comment:
please add test with params as well. Use [this](https://github.com/binier/dgraphium/blob/master/packages/core/test/operators/param-value.test.ts) as an example of how to use params and what to check.

Reply:
done